src/geo/map/torque-layer.js

```diff
@@ -42,10 +39,12 @@ var TorqueLayer = LayerModelBase.extend({
   },
 
   _onAttributeChanged: function () {
-    var reloadVis = _.any(this.ATTRIBUTES_THAT_TRIGGER_VIS_RELOAD, function (attr) {
+    var reloadVis = _.any(ATTRIBUTES_THAT_TRIGGER_VIS_RELOAD, function (attr) {
       if (this.hasChanged(attr)) {
         if (attr === 'cartocss') {
-          return this.previous('cartocss') && this._torqueCartoCSSPropsChanged();
+          var hasTorquePropsChanged = this._torqueCartoCSSPropsChanged();
+          var usesTurboCarto = this._hasTurboCarto();
+          return this.previous('cartocss') && (hasTorquePropsChanged || usesTurboCarto);
```

Comment:
it doesn't matter, it should re-instantiate the map because maps api has to convert that CartoCSS (with turbo-carto) to a valid and readable CartoCSS that torque understands.